cypress-tests/cypress/e2e/PaymentUtils/Novalnet.js

```diff
@@ -0,0 +1,193 @@
+const successfulNo3DSCardDetails = {
+    card_number: "4000000000001091",
+    card_exp_month: "12",
+    card_exp_year: "25",
+    card_holder_name: "Joseph Doe",
+    card_cvc: "123",
+  };
+
+const successfulThreeDSTestCardDetails = {
+  card_number: "4000000000001091",
+  card_exp_month: "12",
+  card_exp_year: "25",
+  card_holder_name: "Joseph Doe",
+  card_cvc: "123",
+  };
+  
+  
+  export const connectorDetails = {
+    card_pm: {
+      PaymentIntent: {
+        Request: {
+          currency: "EUR",
+          customer_acceptance: null,
+          setup_future_usage: "on_session",
+        },
+        Response: {
+          status: 200,
+          body: {
+            status: "requires_payment_method",
+          },
+        },
+      },
+      "3DSManualCapture": {
+        Request: {
+          payment_method: "card",
+          payment_method_data: {
+            card: successfulThreeDSTestCardDetails,
+          },
+          currency: "EUR",
```

Comment:
Currency should only be passed in `PaymentIntent` and no where else. So this needs to be removed.
So, in that case, if currency here is updated to `USD` or something, we get a 4xx for trying to update the currency which is not intended and this is not supported.